externalGravity.cpp

```diff
@@ -173,6 +179,35 @@ Vector3D<double> ExternalGravity::applyPotential(GravityParticle *p) const
         if(idt2 > p->dtGrav)
             p->dtGrav = idt2;
         }
+        
+    if(bLogarithmicHalo) {
+        double px = p->position.x;
+        double py = p->position.y;
+        double pz = p->position.z;
+        double r = p->position.length();
+        // Unit vectors in Cartesian coordinates
```

Comment:
These four lines are not needed any more.

Reply:
I assume you mean lines 188-191 (with the xhat, yhat, and zhat vectors); I think I still need lines 185-187.